examples/src/main/java/com/mapbox/navigation/examples/core/BaseRouterActivityKt.kt

```diff
@@ -0,0 +1,248 @@
+package com.mapbox.navigation.examples.core
+
+import android.content.Context
+import android.os.Bundle
+import android.os.Environment
+import android.widget.Toast
+import androidx.annotation.DrawableRes
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import com.google.android.material.snackbar.Snackbar
+import com.mapbox.api.directions.v5.DirectionsCriteria
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.api.directions.v5.models.RouteOptions
+import com.mapbox.geojson.Point
+import com.mapbox.mapboxsdk.camera.CameraUpdateFactory
+import com.mapbox.mapboxsdk.geometry.LatLng
+import com.mapbox.mapboxsdk.maps.MapboxMap
+import com.mapbox.mapboxsdk.maps.OnMapReadyCallback
+import com.mapbox.mapboxsdk.maps.Style
+import com.mapbox.mapboxsdk.plugins.annotation.SymbolManager
+import com.mapbox.mapboxsdk.plugins.annotation.SymbolOptions
+import com.mapbox.navigation.base.extensions.applyDefaultParams
+import com.mapbox.navigation.base.extensions.coordinates
+import com.mapbox.navigation.base.options.MapboxOnboardRouterConfig
+import com.mapbox.navigation.base.route.Router
+import com.mapbox.navigation.core.accounts.MapboxNavigationAccounts
+import com.mapbox.navigation.examples.R
+import com.mapbox.navigation.examples.utils.Utils
+import com.mapbox.navigation.navigator.MapboxNativeNavigatorImpl
+import com.mapbox.navigation.route.hybrid.MapboxHybridRouter
+import com.mapbox.navigation.route.offboard.MapboxOffboardRouter
+import com.mapbox.navigation.route.onboard.MapboxOnboardRouter
+import com.mapbox.navigation.ui.route.NavigationMapRoute
+import com.mapbox.navigation.utils.extensions.ifNonNull
+import com.mapbox.navigation.utils.network.NetworkStatusService
+import com.mapbox.turf.TurfConstants
+import com.mapbox.turf.TurfMeasurement
+import java.io.File
+import kotlinx.android.synthetic.main.activity_mock_navigation.mapView
+import kotlinx.android.synthetic.main.activity_mock_navigation.newLocationFab
+import timber.log.Timber
+
+abstract class BaseRouterActivityKt : AppCompatActivity(), OnMapReadyCallback,
```

Comment:
Same comments `BaseRouterActivityJava` above apply to `BaseRouterActivityKt`

Reply:
fixed